mods/mc_core/coordinates.lua

```diff
@@ -1,74 +1,106 @@
 function mc_core.record_coordinates(player, message)
-	if mc_core.checkPrivs(player, {interact = true}) then
-		local pmeta = player:get_meta()
-		local pos = player:get_pos()
-		local realmID = pmeta:get_int("realm")
-		local temp, clean_message
-		clean_message = minetest.formspec_escape(message)
-		temp = minetest.deserialize(pmeta:get_string("coordinates")) or {realms = {}, coords = {}, note_map = {}}
+    if mc_core.checkPrivs(player, {interact = true}) then
```

Comment:
I've added the priv check to the callbacks for the "mark" and "share" buttons rather than the coordinate recording function (this one). People without `shout` should still be bale to record coordinates if they'd like, but they should not be able to share the notes associated with them.